.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -17,15 +17,15 @@ body:
         Please search the Ultralytics [Docs](https://docs.ultralytics.com), [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) to see if a similar question already exists.
       options:
         - label: >
-            I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.
+            I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.
```

Comment:
The hypertext links don't appear to render in the preview of this, but I can't see why they wouldn't.

![image](https://github.com/user-attachments/assets/b5349130-f65f-4e10-b374-868ecef8759c)

Reply:
Current "Question Issue" is rendering them, so perhaps it's just an issue with the preview.

![image](https://github.com/user-attachments/assets/894176d8-3256-4060-8cd0-7a0bc1dfe37f)

Reply:
It looks exactly the same, I don't know why it doesn't render!